app/controllers/outbound_webhooks_controller.rb

```diff
@@ -28,6 +44,20 @@ def resources_path
   end
 
   def resource_params
-    super.permit(:stage_id, :url, :username, :password).merge(project: current_project)
+    link = OutboundWebhookStage.new(stage: @stage)
+    super.permit(:url, :username, :password).merge(outbound_webhook_stages: [link])
+  end
+
+  def require_project
+    case action_name
+    when "create"
+      @stage = Stage.find(params[:outbound_webhook].delete(:stage_id))
```

Comment:
This looks backwards

Reply:
stage_id is a fake attribute ... but yeah we can just submit it as a regular param and avoid this delete magic 👍